bot/code_review_bot/tasks/coverity.py

```diff
@@ -53,24 +53,27 @@ class CoverityIssue(Issue):
 
     ANALYZER = COVERITY
 
-    def __init__(self, revision, issue, file_path):
-        self.revision = revision
-        self.path = file_path
+    def __init__(self, analyzer, revision, issue, file_path):
+        super().__init__(
+            analyzer,
+            revision,
+            file_path,
+            line=issue["line"],
+            nb_lines=1,
+            check=issue["flag"],
+            level="error",
+            message=issue["message"],
+        )
         self.reliability = (
             Reliability(issue["reliability"])
             if "reliability" in issue
             else Reliability.Unknown
         )
-        self.line = issue["line"]
         self.build_error = issue.get("build_error", False)
-        self.bug_type = issue["extra"]["category"]
```

Comment:
We don't need this anymore?

Reply:
It's the display name of the check, but it was never displayed on phabricator or mail. Only exported on the debug report, and then not displayed by the frontend...